pymc/ode/__init__.py

```diff
@@ -1,4 +1,8 @@
-"""This submodule contains tools used to perform inference on ordinary differential equations."""
+"""
+This submodule contains tools used to perform inference on ordinary differential equations.
+Due to the nature of the model (as well as included solvers), this may perform slowly.
```

Comment:
```suggestion
This submodule contains tools used to perform inference on ordinary differential equations.

Due to the nature of the model (as well as included solvers), this may perform slowly.
```

Reply:
Change incorporated